azext_edge/edge/providers/support/clusterconfig/billing.py

```diff
@@ -0,0 +1,129 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+from functools import partial
+from typing import Iterable, Optional
+
+from knack.log import get_logger
+
+from ...edge_api import CLUSTER_CONFIG_API_V1, EdgeResourceApi
+from ..base import (
+    DAY_IN_SECONDS,
+    assemble_crd_work,
+    process_cron_jobs,
+    process_deployments,
+    process_jobs,
+    process_replicasets,
+    process_services,
+    process_v1_pods,
+)
+
+logger = get_logger(__name__)
+
+AIO_BILLING_USAGE_NAME_LABEL = "app.kubernetes.io/name in (microsoft-iotoperations)"
+ARC_BILLING_EXTENSION_COMP_LABEL = "app.kubernetes.io/component in (billing-operator)"
+BILLING_RESOURCE_KIND = "billing/"
+
+
+def fetch_pods(
+    since_seconds: int = DAY_IN_SECONDS,
+):
+    # capture billing pods for aio usage
+    billing_pods = process_v1_pods(
+        resource_api=CLUSTER_CONFIG_API_V1,
+        label_selector=AIO_BILLING_USAGE_NAME_LABEL,
+        since_seconds=since_seconds,
+        capture_previous_logs=True,
```

Comment:
`capture_previous_logs` is default true now ...or did you want to be extra explicit?